runtime/compiler/z/codegen/J9TreeEvaluator.cpp

```diff
@@ -9617,7 +9617,7 @@ J9::Z::TreeEvaluator::genArrayCopyWithArrayStoreCHK(TR::Node* node,
 
    // Ready parameter 6: helper reg
    intptr_t *funcdescrptr = (intptr_t*) fej9->getReferenceArrayCopyHelperAddress();
-   if (comp->compileRelocatableCode())
+   if (comp->compileRelocatableCode() || comp->isOutOfProcessCompilation())
```

Comment:
Can't we get the address of the helper from the client here? Since this is constant, it could be saved into `VMInfo` struct that JITServer is maintaining.

Reply:
I take this one back because it's likely you are going to use relative addressing, so even though you know the address of the helper, the position where this code will reside in the end will change, so you need a relocation.